wordpress-mobile/WordPress-iOS
# Stories: iPad Support

Adds iPad support for Stories.

#### Related PRs

Kanvas: https://github.com/tumblr/kanvas-ios/pull/112

## Changes

- Kanvas now supports iPad by fixing rotation and layout bugs.
- Removes to disable Stories for iPad (`EditorCreationError` and other related code)
- Changes the discard action sheet to an alert on iPad
- Uses `confirmButton` to display the prepublishing sheet as a popover.
- Changes the modal presentation style to `overFullScreen` so that it mirrors the iPhone presentation.
- Fixes a resizing bug in the Prepublishing sheet to allow for proper resizing based on the footer contents.

## Testing

- Verify that the "Story post" option is listed in the bottom sheet when tapping the FAB
- Create a Story
- Capture media from your device's camera. Ensure that rotation and capture functionality work as expected.
- Add media from your device's library. Ensure media is properly displayed in a 9:16 aspect ratio centered in the screen.
- Ensure that posting the content works as expected.

PR submission checklist:
